client/reader/following-manage/index.jsx

```diff
@@ -96,10 +92,19 @@ class FollowingManage extends Component {
 			debounce( this.resizeSearchBox, 50 )
 		);
 		this.resizeSearchBox();
+
+		// this is a total hack. In React-Virtualized you need to tell a WindowScroller when the things
+		// above it has moved with a call to updatePosision().  Our issue is we don't have a good moment
+		// where we know that the content above the WindowScroller has settled down and so instead the solution
+		// here is to call updatePosition in a regular interval. the call takes about 0.1ms from empirical testing.
+		this.updatePosition = setInterval( () => {
+			this.windowScrollerRef && this.windowScrollerRef.updatePosition();
+		}, 100 );
```

Comment:
sounds good.

> what content is above the scroller that needs to settle down?
  
The layout of the page from top to bottom is roughly:
1. search bar
2. search results ( up to 3).
3. list of current subscriptions.

number 2 is the one with variable height.